src/components/hero.tsx

```diff
@@ -9,37 +9,46 @@ type HeroProps = {
 };
 
 export function Hero({ allEpisodes }: HeroProps) {
+  if (!allEpisodes || allEpisodes.length < 3) {
+    return null;
```

Comment:
here you can return a message "No episodes found"

https://uxcontentcraft.substack.com/p/content-crit-cest-quoi-un-bon-empty